app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,7 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+        <%= time_ago_in_words(quote.created_at || Time.now) %> ago    
```

Comment:
You should find the null entry and uptate it to have a date. You can just use todays date -- something is better than nothing. 

Also, could you look into adding a constraint on the quotes `active_record` model so that we can't find ourselves in this position again?